js/full-picture.js

```diff
@@ -0,0 +1,56 @@
+import { isEscapeKey } from './utils.js';
+import { DESCRIPTIONS } from './data.js';
+import { getRandomArrayElement } from './utils.js';
+import { createComments } from './comments.js';
+import { commentsArray } from './data.js';
+
+export const interactWithBigPicture = () => {
+  const pictureElement = document.querySelector('.pictures');
+  const bigPictureShow = document.querySelector('.big-picture');
```

Comment:
```suggestion
  const bigPicture = document.querySelector('.big-picture');
```
Это ведь не функция а элемент, глагол не нужен. Не забудь изменить имя во всем файле